src/i-client-config.ts

```diff
@@ -78,6 +78,14 @@ export interface IPrecomputedClientConfig extends IBaseRequestConfig {
   subjectAttributes?: Record<string, AttributeType>;
 }
 
+/**
+ * Configuration for Eppo precomputed client initialization
+ * @public
+ */
+export interface IPrecomputedClientConfig extends IBaseRequestConfig {
```

Comment:
Breaking change to the interface of `precomputedInit` but assuming it is safe because we are currently the only users of this function